drivers/SmartThings/zigbee-thermostat/src/init.lua

```diff
@@ -208,7 +238,9 @@ local do_refresh = function(self, device)
     FanControl.attributes.FanModeSequence,
     FanControl.attributes.FanMode,
     PowerConfiguration.attributes.BatteryVoltage,
-    PowerConfiguration.attributes.BatteryAlarmState
+    PowerConfiguration.attributes.BatteryAlarmState,
+    TemperatureMeasurement.attributes.MinMeasuredValue,
+    TemperatureMeasurement.attributes.MaxMeasuredValue
```

Comment:
These two probably should only be read on Add, since we don't expect them to change. This refresh function is already quite chatty.

In the future, we probably want to do the same with ControlSequenceOfOperation and FanModeSequence, but not in this PR.

Reply:
5d905d794465a528125c7304cb9454af67f29243: moved the two to the `device_added` and updated the tests